### DISCUSSION TITLE
Always display an icon in the notification area?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#14777] [General]
WindowsTerminalPreview: 1.17.2301.23004

I have

![image](https://user-images.githubusercontent.com/61856645/216502736-d45318ef-0f2c-431e-8ac1-75aaf4b9cc59.png)

I'd expect the tray icon to be displayed _only_ when WT is minimized.  But the tray icon is always displayed.  Am I misunderstanding?

**Comment by DHowett** [comment]
I can't read. If you got an email about this comment, know that I was wrong. :D

Your understanding is correct. Looks like a bug!

**Reply by vefatica** [reply]
OK.  I was offered the opportunity to make this an issue ... and I did.

**Reply by vefatica** [reply]
... but I don't see it among the issues.  It's subject is greyed out when I view discussions.  How does that work?

**Comment by CAESIUS-TIM** [comment]
working in `1.22.10731.0`
![image](https://github.com/user-attachments/assets/c54edcda-83f7-4d62-b574-9749f7bd72d1)
